src/NUnitFramework/framework/Internal/MethodWrapper.cs

```diff
@@ -90,6 +90,14 @@ public bool IsPublic
             get { return MethodInfo.IsPublic;  }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the method is static.
+        /// </summary>
+        public bool IsStatic
```

Comment:
Minor, but for short stuff like this, we're trying to update to the new syntax, `public bool IsStatic => Methodinfo.IsStatic;`

Reply:
With pleasure!